clients/unshaded/src/main/java/tachyon/client/ClientContext.java

```diff
@@ -96,7 +96,7 @@ public static synchronized TachyonConf getConf() {
   }
 
   /**
-   * @return the ClientMetrics for this client
+   * @return the {@link ClientMetrics} for this client
```

Comment:
@yupeng9 To clarify, does the current approach sound good, or does the switching to always using full paths sound good? I'd favor the current approach (relative by default, optionally full to avoid adding new imports), since that makes the code comments more readable, and it isn't so bad to occasionally have imports used only by comments.